redisinsight/api/src/modules/database-recommendation/database-recommendation.service.ts

```diff
@@ -88,4 +89,35 @@ export class DatabaseRecommendationService {
     this.logger.log(`Update database extended recommendations id:${id}`);
     return this.databaseRecommendationsProvider.update(clientMetadata, id, dto);
   }
+
+  /**
+   * Delete database recommendation by id
+   * @param clientMetadata
+   * @param id
+   */
+  async delete(clientMetadata: ClientMetadata, id: string): Promise<void> {
+    this.logger.log(`Deleting recommendation: ${id}`);
+    await this.databaseRecommendationsProvider.delete(clientMetadata, id);
+  }
+
+  /**
+   * Bulk delete recommendations. Uses "delete" method and skipping error
+   * Returns successfully deleted recommendations number
+   * @param clientMetadata
+   * @param ids
+   */
+  async bulkDelete(clientMetadata: ClientMetadata, ids: string[]): Promise<DeleteDatabaseRecommendationResponse> {
+    this.logger.log(`Deleting many recommendations: ${ids}`);
+
+    return {
+      affected: sum(await Promise.all(ids.map(async (id) => {
```

Comment:
It is not good practice to delete database entries with multiple queries when it is possible to do with a single one.
We should not have an issue here we will not have a lot of recommendations but keep in mind next time

Reply:
I'll keep keep in mind